kokoro/gcp_ubuntu/bazel/bindings/build_kokoro.sh

```diff
@@ -37,4 +37,7 @@ docker run \
 
 # Kokoro will rsync this entire directory back to the executor orchestrating the
 # build which takes forever and is totally useless.
-rm -rf "${KOKORO_ARTIFACTS_DIR?}/*"
+sudo rm -rf ${KOKORO_ARTIFACTS_DIR?}/*
```

Comment:
Ugh it doesn't do the wildcard expansion if it's in quotes. Ok but we should still quote the variable expansion (https://google.github.io/styleguide/shellguide.html#quoting). `KOKORO_ARTIFACTS_DIR="/ foo"` ;-P

```suggestion
sudo rm -rf "${KOKORO_ARTIFACTS_DIR?}"/*
```